app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -126,7 +126,13 @@ class UrlInputFragment :
 
         // Get session from session manager if there's a session UUID in the fragment's arguments
         arguments?.getString(ARGUMENT_SESSION_UUID)?.let {
-            session = SessionManager.getInstance().getSessionByUUID(it)
+            try {
+                session = SessionManager.getInstance().getSessionByUUID(it)
+            } catch (i: IllegalAccessError) {
+                // When Erase FAB and Url Bar are touched in quick succession, it is possible for
+                // the session we want to not exist anymore, so we can destroy the fragment
+                onDestroy()
+            }
```

Comment:
Oh nice, you can reproduce that!? I haven't been able to do that and that issue has been around for quite some time.

I wouldn't catch the exception here - it's there to be a signal that something is very wrong.

Could we instead ask the SessionManager whether it has a session for this UUID and if not continue without a session (which means there won't be a URL etc. when editing it - but this makes sense when the session was erased.

And also add a big comment here explaining why we do that :)